launch-template.tf

```diff
@@ -80,5 +80,6 @@ resource "aws_launch_template" "worker_nodes" {
     cluster_endpoint           = data.aws_eks_cluster.cluster.endpoint
     certificate_authority_data = data.aws_eks_cluster.cluster.certificate_authority[0].data
     bootstrap_extra_args       = local.bootstrap_extra_args
+    extra_configuration        = var.extra_configuration
```

Comment:
`extra_configuration` is not too meaningful. I think it should be clear this refers to userdata from the launch template.

Perhaps something like `launch_template_custom_userdata`?